lib/active_pivot/activity.rb

```diff
@@ -49,7 +49,12 @@ def primary_resource
     end
 
     def new_values
-      remote_activity.changes[0]['new_values']
+      remote_activity.changes.each do |change|
```

Comment:
That worked great. Thanks! I didn't like the explicit returns in there, `detect` is perfect